### DISCUSSION TITLE
AFK

### DISCUSSION THREAD

**Discussion by danechek** [EssentialsX/Essentials#4053] [Ideas and Feature Suggestions]
Add new commands for AFK info.
For example:
 /afklist - shows list of afk players
/afkinfo <player> - shows how long the player is afk

**Comment by Pheotis** [comment]
The default essentials `/elist` command shows an [AFK] flag besides the name of all afk players.

There is currently no command to show how long a player has been afk for...
Perhaps that info could be included in the `/seen` command, which already includes an entry for `- AFK: true`.

**Reply by DebugOk** [reply]
Essentials doesn't log how long a player is afk for I believe, so adding something like that would require changes to both the AFK system, and the seen command
